src/scenes/Question/scenes/Read/components/Meta/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { DateTime } from 'luxon'
+
+import IconMenu from 'react-toolbox/lib/menu/IconMenu'
+import MenuItem from 'react-toolbox/lib/menu/MenuItem'
+
+import Flag from '../Flag'
+
+class Meta extends Component {
+  render () {
+    const { node, createFlag } = this.props
+
+    const flagIcons = {
+      outdated: 'history',
+      incomplete: 'report'
+    }
+
+    const flags = node.flags.map((flag, i) => {
```

Comment:
Do you have always the same number of flags ? Otherwise, index as key isn't recommended https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
I didn't know about that, I'll look into it, thanks